scripts/ffmprovisr

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env bash
+
+if [[ $OSTYPE = darwin* ]] ; then
+    if [ -d /usr/local/Cellar/ffmprovisr ] ; then
+        ffmprovisr_path=$(find /usr/local/Cellar/ffmprovisr -iname 'index.html' | sort -M | tail -n1)
+    fi
+    if [ -z "${ffmprovisr_path}" ] ; then
+        ffmprovisr_path='https://amiaopensource.github.io/ffmprovisr/'
+    fi
+    open "${ffmprovisr_path}"
+fi
+
+if [[ $OSTYPE = linux-gnu ]] ; then
```

Comment:
There is no law! Please do as you prefer. If there is a reason behind, it’s fine for me.